src/Book/CreateOrEditBook.css

```diff
@@ -0,0 +1,247 @@
+.create-book {
+}
+
+.create-book-title {
+    width: 298px;
+    height: 67px;
+    font-family: PlayfairDisplay, serif;
+    font-size: 50px;
+    font-weight: 900;
+    font-stretch: normal;
+    font-style: italic;
+    line-height: 1.34;
+    letter-spacing: normal;
+    text-align: left;
+    color: #030303;
+    margin: auto;
+}
+
+.create-book-form {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+}
+
+.form-input {
+    width: 300px;
+    height: 50px;
+    border-radius: 18px;
+    border: solid 1px #dfe2e6;
+    background-color: #ffffff;
+}
+
+.rectangle-2 {
+    width: 45%;
+    height: 50px;
+    border-radius: 18px;
+    border: solid 1px #dfe2e6;
+    background-color: #ffffff;
+    margin: 2%;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.input {
+    width: 80%;
+    height: 22px;
+    font-family: Roboto, serif;
+    font-size: 18px;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 2.81;
+    letter-spacing: normal;
+    text-align: left;
+    color: #111111;
+    border: none;
+    margin-top: auto;
+    margin-bottom: auto;
+}
+
+.input:focus, .Password:focus{
+    outline: none;
+
+}
+
+.box {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    margin: 6% 4% 0;
+}
+
+.icon {
+    object-fit: contain;
+    color: #a4a8ad;
+    font-size: x-large;
+    width: 15%;
+    margin-top: auto;
+    margin-bottom: auto;
+    margin-left: 1%;
+}
+
+.rectangle-6 {
+    width: 30%;
+    padding: 2.5%;
+    border-radius: 33px;
+    background-color: #e4e9f0;
+    border: none;
+    margin-top: 5%;
+    margin-bottom: 8%;
+}
+
+.rectangle-6:hover {
+    background-color: #48a3fb;
+}
+
+.rectangle-6:hover .save-button {
+    color: #ffffff;
+}
+
+.save-button {
+    font-family: Roboto, serif;
+    font-size: 20px;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.31;
+    letter-spacing: 0;
+    text-align: center;
+    color: #48a3fb;
+    margin: 0;
+}
+
+.cancel-button {
+    font-family: Roboto, serif;
+    font-size: 20px;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.31;
+    letter-spacing: 0;
+    text-align: center;
+    color: #ffffff;
+    margin: 0;
+}
+
+.rectangle-6-red {
+    width: 30%;
+    padding: 2.5%;
+    border-radius: 33px;
+    background-color: #f44336;
+    border: none;
+    margin-top: 5%;
+    margin-bottom: 8%;
+}
+
+.error-box {
```

Comment:
Centrar el texto del error. Se recomienda usar [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). En este caso bastaria con solo agregar:
`
display: flex;
align-items: center;
justify-content: center;
`